src/build_matrix_A.py

```diff
@@ -19,10 +19,7 @@ def build_matrix_A(material_property_library, mesh, time):
       A.x=b formula
     """
    
-    if len(time) > 1:
-    	Dt = time[1] - time[0]
-    else:
-    	Dt = 1
+
     
     k, rho, c, DR = property_list_builder(material_property_library, mesh)
```

Comment:
It might also be convenient to calculate `k/(rho * c)`?

Reply:
Certainly something worth considering for a future update.